packages/next/src/client/components/react-dev-overlay/internal/helpers/group-stack-frames-by-module.ts

```diff
@@ -1,29 +1,17 @@
 import type { OriginalStackFrame } from './stack-frame'
 
 export type StackFramesGroup = {
-  framework?: 'next' | 'react'
+  moduleGroup?: OriginalStackFrame['sourceModule']
   stackFrames: OriginalStackFrame[]
 }
 
-/**
- * Get the origin framework of the stack frame by package name.
- */
-function getFramework(
-  sourcePackage: string | undefined
-): StackFramesGroup['framework'] {
-  if (!sourcePackage) return undefined
-
-  if (
-    /^(react|react-dom|react-is|react-refresh|react-server-dom-webpack|react-server-dom-turbopack|scheduler)$/.test(
-      sourcePackage
-    )
-  ) {
-    return 'react'
-  } else if (sourcePackage === 'next') {
-    return 'next'
-  }
-
-  return undefined
+/** TODO: Move this to react-dev-overlay/server/middleware.ts */
+function getModuleGroup(file: string | null): StackFramesGroup['moduleGroup'] {
+  if (!file) return
+  // Should we maybe want to group Node.js errors too?
+  // if (file.startsWith('node:internal/')) return 'node'
+  else if (file.startsWith('webpack://next/dist/compiled/react')) return 'react'
```

Comment:
Not sure if this is the best way to detect react, but I believe we started pre-compiling after #44137, so this is how I was able to find the related modules. Is there a better way maybe?